pkgs/tools/misc/ethminer/default.nix

```diff
@@ -0,0 +1,62 @@
+{ fetchurl, lib, pkgs, stdenv, fetchFromGitHub, opencl-headers, cmake, jsoncpp, boost, makeWrapper,
```

Comment:
`pkgs` should never be an argument here, after the change suggested above you can remove it. Also you can remove `lib` and `fetchurl` as these aren't used anywhere.